XamarinCommunityToolkit/Behaviors/TagBehavior.shared.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text.RegularExpressions;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace Microsoft.Toolkit.Xamarin.Forms.Behaviors
+{
+    public class TagBehavior : BaseBehavior<Label>
+    {
+        public IList<TagType> TagTypes { get; } = new List<TagType>();
```

Comment:
Should it be exposed as bindable property?

Reply:
Yes makes sense, just did the change